### PR TITLE
JBPM-5444: Fix SVG issues on some browsers.

### PR DESCRIPTION
**Pull request by romartin**
JBPM-5444: Fix SVG issues on some browsers & added use of picture's onload callbacks.

Depends on:
- https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/329
- https://github.com/uberfire/uberfire/pull/580
- https://github.com/droolsjbpm/drools-wb/pull/359

### CODE REVIEW

**Comment by romartin** [comment]
@manstis this fixes the issues on FF/IE about SVG files. It upgrade lienzo to 2.0.279-RELEASE as well. Notice this lienzo release does not yet contains your CM related commits, but at once a new release with your latest stuff is available, you just need to update our kie-bom, I already tested Stunner with your stuff and seems to work fine and you'll not hit with any API updates.

**Comment by romartin** [comment]
@mdproctor @krisv @hasys these commits are the fixes for the FF/IE issues ^^

**Comment by romartin** [comment]
Hey @manstis ,updated this commit as well for 280-RELEASE. Thanks!

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-shapes/kie-wb-common-stunner-shapes-client/src/main/java/org/kie/workbench/common/stunner/shapes/client/view/PictureShapeView.java [hunk 1]
```diff
@@ -37,9 +37,12 @@ public PictureShapeView( final String uri,
                 .rect( 0, 0, width, height )
                 .setStrokeAlpha( 0 )
                 .setFillAlpha( 0 ) );
-        this.picture = new Picture( uri );
-        scalePicture( picture, width, height );
-        addChild( picture );
+        new Picture( uri, picture1 -> {
```
Comment (jomarko) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-shapes/kie-wb-common-stunner-shapes-client/src/main/java/org/kie/workbench/common/stunner/shapes/client/view/PictureShapeView.java]: Workbench build now fails with this message:
[ERROR] Errors in 'jar:file:/home/jomarko/.m2/repository/org/kie/workbench/stunner/kie-wb-common-stunner-shapes-client/7.0.0-SNAPSHOT/kie-wb-common-stunner-shapes-client-7.0.0-SNAPSHOT-sources.jar!/org/kie/workbench/common/stunner/shapes/client/view/PictureShapeView.java'
[INFO]          [ERROR] Line 40: The constructor Picture(String, (<no type> picture1) -> {}) is undefined
[INFO]          [ERROR] Line 40: The target type of this expression must be a functional interface

Please fix ASAP.

File: kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-shapes/kie-wb-common-stunner-shapes-client/src/main/java/org/kie/workbench/common/stunner/shapes/client/view/glyph/PictureGlyph.java [hunk 2]
```diff
@@ -33,10 +35,17 @@ public PictureGlyph( final String uri,
...
                         final double height ) {
-
-        final Picture picture = new Picture( uri );
-        scalePicture( picture, width, height );
-        group.add( picture );
+        final Rectangle decorator = new Rectangle( width, height )
+                .setCornerRadius( 5 )
+                .setFillColor( ColorName.LIGHTGREY )
+                .setFillAlpha( 0.2d );
+        new Picture( uri, picture -> {
```
Comment (jomarko) [inline kie-wb-common-stunner/kie-wb-common-stunner-client/kie-wb-common-stunner-shapes/kie-wb-common-stunner-shapes-client/src/main/java/org/kie/workbench/common/stunner/shapes/client/view/glyph/PictureGlyph.java]: Hi @romartin .

Workbench build now fails with ths message:
[ERROR] Errors in 'jar:file:/home/jomarko/.m2/repository/org/kie/workbench/stunner/kie-wb-common-stunner-shapes-client/7.0.0-SNAPSHOT/kie-wb-common-stunner-shapes-client-7.0.0-SNAPSHOT-sources.jar!/org/kie/workbench/common/stunner/shapes/client/view/glyph/PictureGlyph.java'
[INFO]          [ERROR] Line 42: The constructor Picture(String, (<no type> picture) -> {}) is undefined
[INFO]          [ERROR] Line 42: The target type of this expression must be a functional interface

Please fix ASAP. Thanks.

**Comment by manstis** [comment]
I compiled ```droolsjbpm-build-bootstrap```, ```uberfire```, ```guvnor``` (although probably not needed) and ```kie-wb-common``` with the latest from ```master``` without issue this morning. I suspect you'll need to rebuild the same (Jenkins was playing up yesterday and some SNAPSHOTs may be old).

**Comment by jomarko** [comment]
@manstis thanks. Problem solved. Anyway, Jenkins still seems to have some troubles.

**Pull request merged**